modules/apps/foundation/frontend-theme/justin-theme/src/js/main.js

```diff
@@ -0,0 +1,104 @@
+AUI().ready(
+
+	/*
+	This function gets loaded when all the HTML, not including the portlets, is
+	loaded.
+	*/
+
+	function(){
+		let i = 0;
+			
+		var button = document.getElementById('btnCollapse');	
+		var text = document.getElementById('btnChild');
+
+		function changeButtonColor(){
+			i++;
+			console.log(i);
+			if(i % 2 == 1){
+				button.style.backgroundColor = '#ff8600';
+				text.style.color = '#FFF';
+			}else{
+				button.style.backgroundColor = '';
+				text.style.color = '';
+				}
+			}
+
+		$("#btnCollapse").click(changeButtonColor);
+	
+
+		// Hide Header on on scroll down
+		var didScroll;
```

Comment:
Use present tense variables if possible.

Reply:
You can rename this to ```documentTop``` or something that checks if it is at the top of the page instead of whether it has scrolled.